<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SKGVisualWidget</class>
 <widget class="QWidget" name="SKGVisualWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frClient">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="frSKG">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="AreaSKG" name="wgtSKG" native="true">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>168</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QTreeView" name="tvFactors"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frTools">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnPlus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../kernel.qrc">
          <normaloff>:/images/Plus.png</normaloff>:/images/Plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMinus">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../kernel.qrc">
          <normaloff>:/images/Minus.png</normaloff>:/images/Minus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZeroing">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../kernel.qrc">
          <normaloff>:/images/Zeroing.png</normaloff>:/images/Zeroing.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AreaSKG</class>
   <extends>QWidget</extends>
   <header location="global">areaskg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>splitter</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>SKGVisualWidget</receiver>
   <slot>splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>820</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlus</sender>
   <signal>clicked()</signal>
   <receiver>SKGVisualWidget</receiver>
   <slot>on_btnPlusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>820</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMinus</sender>
   <signal>clicked()</signal>
   <receiver>SKGVisualWidget</receiver>
   <slot>on_btnMinusClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>822</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnZeroing</sender>
   <signal>clicked(bool)</signal>
   <receiver>SKGVisualWidget</receiver>
   <slot>on_btnZeroingClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>822</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>splitterMoved(int,int)</slot>
  <slot>on_btnPlusClicked()</slot>
  <slot>on_btnMinusClicked()</slot>
  <slot>on_btnZeroingClicked(bool)</slot>
 </slots>
</ui>
